src/main.rs

```
@@ -1,3 +1,28 @@
+use std::collections::HashMap;
+
+const DB_FILENAME: &str = "kv.db";
+
 fn main() {
-    println!("Hello, world!");
+    let database = Database::new().expect("Failed to create database!");
+
+    let mut args = std::env::args().skip(1);
+    let key = args.next().expect("TODO: SHOW USAGE");
```

clippy

unused variable

(warning)